alicloud-test/tests/alicloud_cs_kubernetes_cluster/test-list-query.sql

```diff
@@ -0,0 +1,3 @@
+select name, cluster_id, state
+from alicloud_cs_kubernetes_cluster
+where akas::text = '["{{ output.resource_aka.value }}"]' and region = '{{ output.region.value }}';;
```

Comment:
```suggestion
where akas::text = '["{{ output.resource_aka.value }}"]' and region = '{{ output.region.value }}';
```

Reply:
@bigdatasourav Is region needed to list?

Reply:
yes